Sources/UrlFormEncoding/UrlFormDecoder.swift

```diff
@@ -324,7 +324,7 @@ public final class UrlFormDecoder: Decoder {
     }
 
     mutating func decode(_ type: Bool.Type) throws -> Bool {
```

Comment:
I think we can avoid some of this repetition, but my first stab didn't work. Think the nested container logic may not be completely right.